### DISCUSSION TITLE
installing Mxnet==1.8 on windows

### DISCUSSION THREAD

**Discussion by mohamadHN93** [apache/incubator-mxnet#20322] [Q&A]
I would be grateful if you help me in installing the mxnet vesion 1.8.0 on pycharm on windows since it is not possible to be caught through pip install.

**Comment by TristonC** [comment]
Haven't tried pycharm yet. Is that the only option for you to install on windows? Could it be Window's wsl or conda?

**Reply by TristonC** [reply]
@mnakhaeecourse Do you need GPU support?

**Reply by TristonC** [reply]
I have no problem installing MXNet 1.8.0 on windows 10 with WSL and anaconda.

**Reply by mohamadHN93** [reply]
I couldn't install in on windows. now I wanted to install it on google colab, can you help me
when I use this command, it faced this error
!pip install https://github.com/apache/incubator-mxnet/archive/refs/tags/1.8.0.tar.gz
or
!python -m pip install https://github.com/apache/incubator-mxnet/archive/refs/tags/1.8.0.tar.gz
I got the below error:
Collecting https://github.com/apache/incubator-mxnet/archive/refs/tags/1.8.0.tar.gz
  Using cached https://github.com/apache/incubator-mxnet/archive/refs/tags/1.8.0.tar.gz
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Reply by Adnios** [reply]
Hi @mcourse1993 . I think you can install mxnet in google colab easily using command like
```
!pip install mxnet==1.8.0
```
And I don't think command `!pip install https://github.com/apache/incubator-mxnet/archive/refs/tags/1.8.0.tar.gz` is working for mxnet. Because there is no setup.py in mxnet directory(setup.py is exited in `mxnet/python/` directory). If you want to install mxnet from source, you should build first and `cd python && pip install --user -e `
